### PR TITLE
Change license to Apache 2.0

### LINKED ISSUE #194: license change to Apache-2.0
Short version:
Hi giter8 contributors! Can I get your ok on changing the license to Apache-2.0? Please comment "yes, I approve" or "no".

@hadesgames, @peutetre, @keshin, @pk11, @jto

yes:
@xuwei-k, @softprops, @joescii, @thirstycrow, @ches, @butlermh, @pvlugter, @njlg, @KirinDave, @jberkel, @tlockney

Long version:
For wider adoption of giter8, I think it would help if we can switch the license from LGPL to Apache-2.0. I've gotten ok from Nathan (@n8han) but there are a dozen more contributors who has contributed 10 lines or more. I'm not sure how fast we can get everyone's ok, but if we can pull off the switch, I think it would be great.

### PR DESCRIPTION
**Pull request by eed3si9n**
Fixes #194

This change reverts any commits that I wasn't get confirmation in #194 or via private correspondence.

### CODE REVIEW

**Comment by padilo** [comment]
👍 I found a reference to n8han/giter8 repository but I think is out of scope. I'll pin it anyway

File: app/src/main/scala/giter8.scala [hunk 1]
```diff
@@ -68,9 +75,6 @@ class Giter8 extends xsbti.AppMain with Apply {
       |Apply template from a remote branch
       |    g8 n8han/giter8 -b some-branch
```
Comment (padilo) [inline app/src/main/scala/giter8.scala]: This should be ```g8 foundweekends/giter8 -b some-branch```.

Comment (eed3si9n) [inline app/src/main/scala/giter8.scala]: Could you open a new issue on this?

Comment (eed3si9n) [inline app/src/main/scala/giter8.scala]: Made a new issue - https://github.com/foundweekends/giter8/issues/196

**Comment by fommil** [comment]
:cry: it's sad to see software lose its copyleft status...

**Comment by eed3si9n** [comment]
@fommil My wager is that this change would be net positive for the viability of both the giter8 the program, as well as giter8 the idea: Github-hosted templating system. Plus there's little risk of AT&T making a proprietary OS out of it. Giter8's not UNIX®.

**Pull request merged**